### PR TITLE
bibliothecary 6.9.7 ~> 6.12.2

### PR DESCRIPTION
**Pull request by tiegz**
Looks like we haven't bumped bibliothecary in a few months, so here's a diff:

https://github.com/librariesio/bibliothecary/compare/v6.9.7...v6.12.2

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ ruby "2.6.5"
 gem "active_model_serializers"
 gem "api-pagination"
 gem "asciidoctor"
-gem "bibliothecary"
+gem "bibliothecary", "~> 6.12.2"
```
Comment (jsonperl) [inline Gemfile:9]: Any reason to specify rather than just a bundle update.

**Review by jsonperl — APPROVED** [review]

File: Gemfile (hunk 1 above)
Comment (tiegz) [reply Gemfile:9]: Strangely that wasn't updating the gem for me! I wonder if it's because it's a new minor version?

Comment (jsonperl) [reply Gemfile:9]: May have been that rubygems didn't have it yet?

Comment (tiegz) [reply Gemfile:9]: Wait a sec, I tried again and it worked. I bet I was just running `bundle` before 🤦🏻

**Review by jsonperl — APPROVED** [review]

**Pull request merged**